File: py/product.py
class Product:
    def __init__(self, idProduct, name, brand, price):
        self.idProduct = idProduct
        self.name = name
        self.brand = brand
        self.price = price

    def toTableRow(self):
        return f"| {self.idProduct:<2} | {self.name:<12} | {self.brand:<8} | {self.price:<10} |"

File: cpp/Product.cpp
// Import Library
#include <iostream>
#include <string>

using namespace std;

class Product
{
    protected:
        int idProduct;
        string name;
        string brand;
        string price;

    public:
        Product(int idProduct, string name, string brand, string price)
        {
            this->idProduct = idProduct;
            this->name = name;
            this->brand = brand;
            this->price = price;
        }

        string toTableRow()
        {
            return "| " + to_string(idProduct) + " | " + name + " | " + brand + " | " + price + " |";
        }
};